pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -1,5 +1,5 @@
-Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.32.5 (iks-clusterid:1[192.168.32.5]) within ky-test-vpc
-=================================================================================================================================================
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LoadBalancer] to 192.168.32.5 (iks-clusterid:1[192.168.32.5]) within ky-test-vpc
```

Comment:
Same comment as above. The header should reflect the input queried for **src** (and dst)

Reply:
1. If the src/dst is internal and contains more than a single item then print all the items in brackets, as done in this PR (like we do for internal CIDR at the moment)
2.  If the src/dst is internal and contains a single element - add the type of the element type VSI/LoadBalancer etc in brackets. 

At the moment I have a somehow clumsy mechanism for printing the endpoints of an internal CIDR. The proposed solution will also replace this mechanism. @adisos ?

Reply:
After a short brainstorming we agreed to keep it simple, and have two lines for the header.
```
Explaining connectivity from <input src> to <input dst> .... (as now, without details per internal address of src/dst)
Interpreted Src:  <src details>
Interpreted Dst:  <dst details> 
```
for external node, no need to detail the nodes list , just repeat the input address